src/GraphQL/Builders/ConnectionBuilder.cs

```diff
@@ -222,27 +248,101 @@ public ConnectionBuilder<TSourceType> Argument<TArgumentGraphType>(string name,
         /// <param name="name">The name of the argument.</param>
         /// <param name="description">The description of the argument.</param>
         /// <param name="defaultValue">The default value of the argument.</param>
-        public ConnectionBuilder<TSourceType> Argument<TArgumentGraphType, TArgumentType>(string name, string description,
-            TArgumentType defaultValue = default)
+        public ConnectionBuilder<TSourceType> Argument<TArgumentGraphType, TArgumentType>(string name, string? description,
+#if NETSTANDARD2_1
+            [AllowNull]
```

Comment:
🤔 Try to remove this and just use `TArgumentType? defaultValue = default`. It should work for C#9.

Reply:
It compiles, but does not provide the same semantics as the prior code.  If `TArgumentType` is `int`, then it will become `int? defaultValue = null` rather than `int defaultValue = default`.  Perhaps this is more desirable, but in any case, it is different (and has a different method signature), and so it is a breaking change.

I would suggest changing both the field and connection builders for v5 to use `TArgumentType?` so that no default is assigned to the argument if the argument is not specified.  But for v4, I would just add the appropriate NRT annotations and keep the semantics uniform.

Reply:
Ok